exercises/rna-transcription/example/RNA.ceylon

```diff
@@ -0,0 +1,13 @@
+class Error() {}
+
+String|Error transcription(String dna) {
+  return dna.fold("" of String|Error)((String|Error r,
+                                       Character    c) =>   if(is Error r)
```

Comment:
Basically, it just aligns the first parenthesis with the first parentheses of the `else` and `then`. It makes it easier for the eye to see what the arguments are for each branch of the `if` statement, to me. I can change it, if you'd prefer.